virtualBridge.py

```diff
@@ -73,8 +73,8 @@ def filter_dhcp_leases(j: list[dict[str, str]], removed_macs: list[str], names:
         xml_str = self.hostconn.run("virsh net-dumpxml default").out
         q = et.fromstring(xml_str)
         removed_macs = []  # type: list[str]
-        names = [vm.name for vm in vms]
-        ips = [vm.ip for vm in vms]
+        names = [vm.name for vm in nodes]
```

Comment:
Nit, they're not necessarily "vms" right?

Reply:
Done